src/phoenix/constants.js

```diff
@@ -1,23 +1,18 @@
 /*
- * Copyright (c) 2021 - present core.ai . All rights reserved.
+ * GNU AGPL-3.0 License
  *
- * Permission is hereby granted, free of charge, to any person obtaining a
- * copy of this software and associated documentation files (the "Software"),
- * to deal in the Software without restriction, including without limitation
- * the rights to use, copy, modify, merge, publish, distribute, sublicense,
- * and/or sell copies of the Software, and to permit persons to whom the
- * Software is furnished to do so, subject to the following conditions:
+ * Modified work Copyright (c) 2021 - present Core.ai
```

Comment:
Use this copyright string as this part is created by phoenix project.
`Copyright (c) 2021 - present core.ai . All rights reserved.`
Same for other files in the phoenix folder. 

Reatin all copyright strings as is in files.